src/components/Calculator.js

```diff
@@ -1,40 +1,78 @@
 import React from 'react';
 import './Calculator.css';
+import { calculate, isNumber } from '../logic/calculate';
 
 class Calculator extends React.PureComponent {
   constructor(props) {
     super(props);
-    this.state = '';
+    this.state = { toDisplay: 0 };
+    this.obj = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    this.buttonsEvt = {
+      AC: 'AC',
+      plusminus: 'X/-',
+      percent: '%',
+      seven: '7',
+      eight: '8',
+      nine: '9',
+      four: '4',
+      five: '5',
+      six: '6',
+      one: '1',
+      two: '2',
+      three: '3',
+      zero: '0',
+      decimal: '.',
+      divide: '÷',
+      multiply: 'x',
+      substract: '-',
+      add: '+',
+      equal: '=',
+    };
   }
 
-  render() {
-    return (
-      <div className="container">
-        <div className="keypad">
-          <input className="div1" type="number" placeholder="0" />
-          <button type="button">AC</button>
-          <button type="button">+/-</button>
-          <button type="button">%</button>
```

Comment:
- [ ] It seems these two buttons are not functioning as expected. For example, the `+/-` button should make the value negative or positive. As per your implementation, the value doesn't change at all. Kindly fix this.

Also, the `%`  button should work as modulus. As per your implementation, there is no visible change to the value when you click this button. Kindly fix this too.

Reply:
@peterrobert Hello! thank you for the review, I fixed the '+/-' button but the '%' it's impossible for me, I don't know how to fix it, and I saw a lot of people which have approved this project without that button working and they are not able to help me, can you help me, please?